snapcraft/common.py

```diff
@@ -68,25 +68,30 @@ def run_output(cmd, **kwargs):
                                        **kwargs).decode('utf8').strip()
 
 
-_DEB_TRANSLATIONS = {
+_ARCH_TRANSLATIONS = {
     'armv7l': {
-        'arch': 'armhf',
+        'kernel': 'arm',
+        'deb': 'armhf',
```

Comment:
What, how is that possible? :smile:

If everyone agreed we'd be jobless :wink: